docs/julia.rst

```diff
@@ -248,8 +250,14 @@ We will use a destructuring notation for the AST nodes.
         G, L, break
     E(G, L, continue: BreakContinue) =
         G, L, continue
+    E(G, L, <if condition body>: If) =
+        let G0, L0, v = E(G, L, condition)
+        if v is true or non-zero:
```

Comment:
This part is about julia, I think that is more strict and only considers `if v is true`.